soh/soh/Enhancements/randomizer/randomizer_check_objects.cpp

```diff
@@ -884,57 +885,57 @@ void RandomizerCheckObjects::UpdateImGuiVisibility() {
             (!RandomizerCheckObjects::AreaIsDungeon(locationIt.rcArea) ||
                 locationIt.vOrMQ == RCVORMQ_BOTH ||
                 locationIt.vOrMQ == RCVORMQ_MQ &&
-                    ((CVarGetInteger("gRandomizeMqDungeons", RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_SET_NUMBER && (CVarGetInteger("gRandomizeMqDungeonCount", 12) > 0) || // at least one MQ dungeon
-                      CVarGetInteger("gRandomizeMqDungeons", RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_RANDOM_NUMBER)) ||
+                    ((CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_SET_NUMBER && (CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeonCount"), 12) > 0) || // at least one MQ dungeon
+                      CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_RANDOM_NUMBER)) ||
                 locationIt.vOrMQ == RCVORMQ_VANILLA &&
-                    (CVarGetInteger("gRandomizeMqDungeons", RO_MQ_DUNGEONS_NONE) != RO_MQ_DUNGEONS_SET_NUMBER || CVarGetInteger("gRandomizeMqDungeonCount", 12) < 12) // at least one vanilla dungeon
+                    (CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) != RO_MQ_DUNGEONS_SET_NUMBER || CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeonCount"), 12) < 12) // at least one vanilla dungeon
```

Comment:
As the cvars are already getting renamed, I think this would be a good change.
```diff
-                    ((CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_SET_NUMBER && (CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeonCount"), 12) > 0) || // at least one MQ dungeon
-                      CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_RANDOM_NUMBER)) ||
-                locationIt.vOrMQ == RCVORMQ_VANILLA &&
-                    (CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeons"), RO_MQ_DUNGEONS_NONE) != RO_MQ_DUNGEONS_SET_NUMBER || CVarGetInteger(RANDOMIZER_SEED_CVAR("MqDungeonCount"), 12) < 12) // at least one vanilla dungeon
+                    ((CVarGetInteger(RANDOMIZER_SEED_CVAR("MQDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_SET_NUMBER && (CVarGetInteger(RANDOMIZER_SEED_CVAR("MQDungeonCount"), 12) > 0) || // at least one MQ dungeon
+                      CVarGetInteger(RANDOMIZER_SEED_CVAR("MQDungeons"), RO_MQ_DUNGEONS_NONE) == RO_MQ_DUNGEONS_RANDOM_NUMBER)) ||
+                locationIt.vOrMQ == RCVORMQ_VANILLA &&
+                    (CVarGetInteger(RANDOMIZER_SEED_CVAR("MQDungeons"), RO_MQ_DUNGEONS_NONE) != RO_MQ_DUNGEONS_SET_NUMBER || CVarGetInteger(RANDOMIZER_SEED_CVAR("MQDungeonCount"), 12) < 12) // at least one vanilla dungeon
```

Reply:
Huh, wonder how I missed that. Also, yeah, that's why I pinged Proxy in Discord asking clarification on that, but he appears to be away for a while.

Reply:
Wait, what are you actually suggesting here? GitHub is not being very clear.

Reply:
Ohhh, I see. MQ instead of Mq. Fair point.